install/steps/3-requirements.php

```diff
@@ -26,17 +26,19 @@
 function version_check($name, $ok, $info = '', $warning = false)
 {
 	global $failed;
-	echo '<p class="' . ($ok ? 'success' : ($warning ? 'warning' : 'error')) . '">' . $name;
+	echo '<div class="alert alert-' . ($ok ? 'success' : ($warning ? 'warning' : 'danger')) . '">' . $name;
 	if(!empty($info))
 		echo ': <b>' . $info . '</b>';
 
-	echo '</p>';
+	echo '</div>';
 	if(!$ok && !$warning)
 		$failed = true;
 }
 
 $failed = false;
 
+echo ($failed) ? '<div class="alert alert-warning"><span>' . $locale['step_requirements_failed'] . '</span></div>' : '';
```

Comment:
This needs to be moved to the line 75 (after if($failed))

It's about following message
![image](https://user-images.githubusercontent.com/16687302/102701141-35aabc80-4254-11eb-9ceb-6e12a5145131.png)